views/album-tracks.hbs

```diff
@@ -0,0 +1,5 @@
+{{#each results}}
+    <img src="{{images.0.url}}" alt="" />
+    <h2>{{name}}</h2>
+    <a href="/albums/{{id}}">view albums</a>
```

Comment:
Instead of a button sending you back, you should have a player for the sample of the song